lib/happo/views/diffs.erb

```diff
@@ -19,22 +19,26 @@
     </header>
 
     <main class="main">
-      <h2>Diffs (<%= diff_images.count %>)</h2>
-      <% diff_images.each do |diff| %>
-        <h3>
-          <%= Rack::Utils.escape_html(diff[:description]) %> @ <%= diff[:viewport] %>
-        </h3>
-        <p><img src="<%= diff[:url] %>"></p>
+      <% if diff_images.count > 0%>
+        <h2>Diffs (<%= diff_images.count %>)</h2>
+        <% diff_images.each do |diff| %>
+          <h3>
+            <%= Rack::Utils.escape_html(diff[:description]) %> @ <%= diff[:viewport] %>
+          </h3>
+
+          <img src="<%= diff[:url] %>">
+        <% end %>
       <% end %>
 
-      <hr>
+      <% if new_images.count > 0%>
+        <h2>New examples (<%= new_images.count %>)</h2>
+        <% new_images.each do |image| %>
+          <h3>
+            <%= Rack::Utils.escape_html(image[:description]) %> @ <%= image[:viewport] %>
+          </h3>
 
-      <h2>New examples (<%= new_images.count %>)</h2>
-      <% new_images.each do |image| %>
-        <h3>
-          <%= Rack::Utils.escape_html(image[:description]) %> @ <%= image[:viewport] %>
-        </h3>
-        <p><img src="<%= image[:url] %>"></p>
+          <img src="<%= image[:url] %>">
+        <% end %>
```

Comment:
I had thought about doing this before adding the count to the header. And after that it didn't feel like it was needed. I'm okay with this anyway. The `<hr>` removal was a good move.